setup.py

```diff
@@ -35,7 +35,7 @@ def find_version(fname):
 history = open('HISTORY.rst').read().replace('.. :changelog:', '')
 requirements = [
     str(requirement.req)
-    for requirement in parse_requirements('requirements.txt')
+    for requirement in parse_requirements('requirements.txt', session=uuid.uuid1())
```

Comment:
Why are you using uuid.uuid1() as the session? It should be a pip.download.PipSession().